docs/batch_inference_with_ts.md

```diff
@@ -106,7 +106,7 @@ $ curl -X POST "localhost:8081/models?url=resnet-152.mar&batch_size=8&max_batch_
 $ curl localhost:8081/models/resnet-152
 {
   "modelName": "resnet-152",
-  "modelUrl": "https://s3.amazonaws.com/model-server/model_archive_1.0/examples/resnet-152-batching/resnet-152.mar",
+  "modelUrl": "https://endpoint/model-server/model_archive_1.0/examples/resnet-152-batching/resnet-152.mar",
```

Comment:
i suggest to change it to
"modelUrl": "<your_model_path>",

Reply:
I think we should it as this way, just modify the endpoint part, or else user will confuse which model.mar to use